processor/spanmetricsprocessor/processor_test.go

```diff
@@ -323,9 +365,28 @@ func newProcessorImp(mexp *mocks.MetricsExporter, tcon *mocks.TracesConsumer, de
 	}
 }
 
+// verifyConsumeMetricsInputCumulative expects one accumulation of metrics, and marked as cumulative
+func verifyConsumeMetricsInputCumulative(t testing.TB, input pdata.Metrics) bool {
+	return verifyConsumeMetricsInput(t, input, pdata.MetricAggregationTemporalityCumulative, 1)
+}
+
+// verifyConsumeMetricsInputDelta expects one accumulation of metrics, and marked as delta
+func verifyConsumeMetricsInputDelta(t testing.TB, input pdata.Metrics) bool {
+	return verifyConsumeMetricsInput(t, input, pdata.MetricAggregationTemporalityDelta, 1)
+}
+
+var numCumulativeConsumptions = 0
```

Comment:
A global var within a test file should either have a comment explaining its existence or moved to be scoped locally within a test function.

Could I ask you to do one or the other?

Reply:
Changed it to local scope